lib/Adafruit_BNO055/Adafruit_BNO055.cpp

```diff
@@ -90,8 +88,7 @@ bool Adafruit_BNO055::begin(adafruit_bno055_opmode_t mode)
     //delay(100);
   //}
   //delay(50);
-
-  DEBUG("rebooted");
+  //DEBUG("rebooted");
```

Comment:
we should just remove this I think.

Reply:
If you do not need your remarks anymore, I can delete them.
But added you as collaborator, so you can make modifications too, if you want.